### ISSUE TITLE
Add visible link to RSS Feed

### ISSUE THREAD

**Issue by bmuenzenmeyer** [nodejs/nodejs.org#5477]
Evaluate if the site could use a visible RSS / Feed link.

I didn't see it on the current site or anywhere on nodejs.dev

It could aide quick debugging efforts too as its an important source of information. We could also research if the vercel preview could generate a preview link handy in each PR

@ovflowd told me that the information is currently only in the head 

<img width="852" alt="image" src="https://user-images.githubusercontent.com/298435/251453899-2f3ebbf9-270e-42ea-a0ca-ce6636cb3f5a.png">

---

_Originally posted by @bmuenzenmeyer in https://github.com/nodejs/nodejs.org/pull/5475#pullrequestreview-1516302074_

**Comment by ovflowd** [comment]
What sort of suggestions would you recommend?

**Comment by bmuenzenmeyer** [comment]
I initially thought it could go in the footer: my rationale being that a user looking for RSS feeds will `cmd+f RSS` and it will show up. 

![image](https://github.com/nodejs/nodejs.org/assets/298435/551f811b-2b78-43bd-ad30-a6a158ccc63f)

I think there are two problems with this, however:

- there is more than one RSS feed
- the footer seems wholly devoted to OpenJS Foundation information, not Node.js

---

So, understanding that https://nodejs.org/en/blog will change, and that we have multiple feeds to work with anyways...the next most obvious place to consider is the nodejs.dev blog pages themselves. They are already broken out by category.

So something like this, added to the category description.

![image](https://github.com/nodejs/nodejs.org/assets/298435/9737383a-c913-4330-be96-91007e0b0344)

This doesn't get people the `cmd+f` discovery, but it is a small hint for those that are seeking this info. I would have never thought to look in the head for this. Is that a standard or best practice?

This is obviously a low priority, and is contingent on the timing of the blog's migration.

**Comment by ovflowd** [comment]
The green icon on the right for the redesign looks like a good candidate. Let's consider doing this change for the Website Redesign.

As mentioned before, we want to avoid making layout changes on nodejs.org at this point.

**Comment by ovflowd** [comment]
@bmuenzenmeyer I forgot to mention but where would we add this to the new redesign?

**Comment by bmuenzenmeyer** [comment]
Can we ask Hayden??

**Comment by ovflowd** [comment]
> Can we ask Hayden??

Feel free to do so :)

**Comment by bmuenzenmeyer** [comment]
asked in Slack. this issue is awaiting his direction then

**Comment by AugustinMauroy** [comment]
@bmuenzenmeyer any news ?

**Comment by bmuenzenmeyer** [comment]
The figma was updated but I don't think it is aligned with the tabbed structure. I'll look when I'm not mobile.
